CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Development
+
+This doc explains how to set up a development environment so you can get started contributing to this project
+
+## Prerequisites
+
+1. [`go`](https://golang.org/doc/install): For building the project
+1. [`git`](https://help.github.com/articles/set-up-git/): For source control
+1. [`gcloud`](https://cloud.google.com/sdk/docs/install): For Interacting with the Google Cloud Platform
+
+This guides breaks down into two parts 
+1. Setting up the GKE cluster - If you have already setup the GKE cluster, you can skip this part
+2. Setting up the development environment - This part is required for all the contributors
+
+### Setting up the GKE cluster
+
+1. First Go ahead and create new project in the Google Cloud Platform and enable billing for the project if you don't have it already enabled. You can follow the below link to create a new project
+[Creating projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+
+
+2. Once you have created the project enable the below apis 
+
+    ```
+    gcloud services enable compute.googleapis.com
+    gcloud services enable container.googleapis.com
+    ```
+
+3. Next go ahead and create the service account in the project and assign the following role to the service account
+
+    - Kubernetes Engine Admin
+    - Monitoring Admin
+    - Service Account User
+
+
+4. Once you have assigned the roles to the service account, go ahead and create a new key for the service account by following this link [Creating and managing service account keys](https://cloud.google.com/iam/docs/creating-managing-service-account-keys)
+
+
+> Note : Make sure you have the service account key stored in a secure location and do not share it with anyone.
+
+5. Now configure the gcloud sdk to use the service account key by running the below command
+
+    ```
+    gcloud auth activate-service-account --key-file /path/to/key.json
+    ```
+
+6. Next go ahead and create a new cluster in the project by running the below command
+
+    ```bash
+    export PROJECT_ID=<project-id>
+    export CLUSTER_NAME=<cluster-name>
+    export REGION=<region>
```

Comment:
We can combine the two parts into one.
```bash
    export PROJECT_ID=<project-id>
    export CLUSTER_NAME=<cluster-name>
    export REGION=<region>

    gcloud container clusters create $CLUSTER_NAME --location=$REGION --num-nodes 1 --machine-type=n1-standard-1 --disk-size=10 --project $PROJECT_ID 
```

Reply:
Any reason why you want to make it a single command, I understand it will be make user life easier but I think it will be better if we segerate as each command servers different purpose

Reply:
If we break the steps into separate parts, we can add more details as follows:  

First, configure the necessary environment parameters:
   ```  
   xxx  
   ```  

Next, create a GKE cluster with the following command:
   ```  
   xxx  
   ```

Reply:
This is also done